drivers/backup/portworx/portworx.go

```diff
@@ -334,6 +334,22 @@ func (p *portworx) GetClusterName(ctx context.Context, orgID string, clusterUid
 	return "", fmt.Errorf("cluster with uid '%s' not found for org '%s'", clusterUid, orgID)
 }
 
+func (p *portworx) GetClusterStatus(orgID string, clusterName string, ctx context.Context) (api.ClusterInfo_StatusInfo_Status, error) {
+	clusterEnumerateReq := &api.ClusterEnumerateRequest{
+		OrgId: orgID,
+	}
+	enumerateRsp, err := p.EnumerateCluster(ctx, clusterEnumerateReq)
+	if err != nil {
+		return api.ClusterInfo_StatusInfo_Invalid, err
+	}
+	for _, cluster := range enumerateRsp.GetClusters() {
+		if cluster.GetName() == clusterName {
+			return cluster.Status.Status, nil
+		}
+	}
+	return api.ClusterInfo_StatusInfo_Invalid, fmt.Errorf("cluster with name '%s' not found for org '%s'", clusterName, orgID)
```

Comment:
Same here. Return nil

Reply:
nil is not a subtype of api.ClusterInfo_StatusInfo_Status